Add another example for TTS notifications (#400)
home-assistant/companion.home-assistant: docs/notifications/basic.md

          channel: alarm_stream_max
```

> If you do not want the TTS notification to be spoken when the Ringer mode is not `normal` (to mimic the behavior of classic notifications), you need to prevent the TTS notification to be sent in the first place. This can be done by adding a condition in your automation that checks the state of another sensor, like the Ringer Mode (`audio_sensor`) or the Audio Mode (`audio_mode`).

Reviewer: So we actually renamed `audio_sensor` to `ringer_mode` when we did the sensor breakup so to new users they will see `ringer_mode`. Also `audio_mode` is not the correct sensor for this as it does not relate to the actual ringer mode.

Lets also add a link to the sensor as well so they know its part of the app.  

https://companion.home-assistant.io/docs/core/sensors#audio-sensors

Author: Don't you think both sensors are relevant for this use case? I think it could be annoying to have the TTS speaks when the phone is ringing or even worse when you are in a call. That why using the `audio_mode` to check if the phone is "idle" (state is `normal`) makes sense to conditionally send the TTS command. That's just another idea, then people can check the documentation to understand the difference.

```suggestion
If you do not want the TTS notification to be spoken when the Ringer mode is not `normal` (to mimic the behavior of classic notifications), you need to prevent the TTS notification to be sent in the first place. This can be done by adding a condition in your automation that checks the state of the [Ringer Mode](https://companion.home-assistant.io/docs/core/sensors#audio-sensors) (`ringer_mode`). The [Audio Mode](https://companion.home-assistant.io/docs/core/sensors#audio-sensors) (`audio_mode`) can also be checked to prevent the TTS to speak when you are in a call.
```

Reviewer: Ok makes sense :) lets just keep 1 link since it goes to the same place and drop the duplicate sensor names since it looks odd next to each other.  The audio sensors are indeed a good use here.
